### PR TITLE
(pup-7422) add deviceconfig parameter to help

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Comment by tphoney** [comment]
@DavidS fancy a merge ?

File: lib/puppet/application/device.rb [hunk 1]
```diff
@@ -77,10 +77,10 @@ def help
 
 USAGE
 -----
-  puppet device [-d|--debug] [--detailed-exitcodes] [-V|--version]
+  puppet device [-d|--debug] [--detailed-exitcodes] [--deviceconfig <file>]
```
Comment (joshcooper) [inline lib/puppet/application/device.rb]: Since deviceconfig takes an optional parameter, can you update the usage to show that?

Comment (tphoney) [inline lib/puppet/application/device.rb]: is there other documentation i can borrow from ? @joshcooper

Comment (joshcooper) [inline lib/puppet/application/device.rb]: Ah, nevermind, what you have is fine. For some reason, I thought it just said `[--deviceconfig]`

**Comment by joshcooper** [comment]
Just a nit, we generally use uppercase in the git commit summary, e.g. `(PUP-XXX)`. Also could you update the commit summary with what deviceconfig does? I know it has something to do with `puppet device`, but beyond that IDK. Do settings in device config override what's in puppet.conf? Are they completely different settings?

**Comment by DavidS** [comment]
@joshcooper see https://docs.puppet.com/puppet/latest/configuration.html#deviceconfig for more info

**Comment by joshcooper** [comment]
@DavidS Linking to docs in the PR won't help git history. Please include some description in the commit summary as described in https://github.com/puppetlabs/puppet/blob/master/CONTRIBUTING.md

**Pull request merged**